### PR TITLE
28 expose apis from met

### PR DESCRIPTION
**Pull request by kstigen**

### CODE REVIEW

File: app/weather_api/services/met_service.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            async with session.get(
+                f"{settings.met_api_url}/sunrise/3.0/sun",
+                params={"lat": lat, "lon": lon, "date": date},
+            ) as response:
+                return TypeAdapter(METJSONSunrise).validate_python(
+                    await response.json()
+                )
+
+    def health_check(self) -> tuple[bool, str]:
+        if (
```
Comment (dymbe) [inline app/weather_api/services/met_service.py:36]: Suggestion: would this be more clean if you split this up a bit? I.e.

```
response = requests.(f"{settings.met_api_url}/locationforecast/2.0/healthz")
if response.status_code == 200
    (....)
```

File: app/weather_api/services/met_service.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import aiohttp
+import requests
```
Comment (dymbe) [inline app/weather_api/services/met_service.py:2]: Why are you using both aiohttp and requests? Preferable you should only use aiohttp (or another async-capable library, like httpx), as requests is not async

File: pyproject.toml [hunk 3]
```diff
@@ -8,16 +8,19 @@ packages = [{include = "weather_api", from = "app"}]
...
 python = "^3.11"
-fastapi = "^0.99.1"
-uvicorn = "^0.22.0"
 black = "^23.3.0"
 py-healthcheck = "^1.10.1"
-pydantic = "1.10.11"
+pydantic = "^2.4.2"
+fastapi = "^0.103.2"
+uvicorn = "^0.23.2"
+aiohttp = "^3.8.6"
```
Comment (dymbe) [inline pyproject.toml:16]: If you do in fact want to use `requests`, you should also include it here

**Review by dymbe — CHANGES_REQUESTED** [review]
Looks good overall, I just have a few suggestions

File: app/weather_api/routes/system_resources.py [hunk 4]
```diff
@@ -3,13 +3,14 @@
...
+from weather_api.dependencies import get_met_service
+from fastapi import Depends
 
 router = APIRouter()
 
 
-@router.get(
-    "/",
-)
+@router.get("/")
```
Comment (dymbe) [inline app/weather_api/routes/system_resources.py:13]: Is this route neccessary?

Comment (kstigen) [reply app/weather_api/routes/system_resources.py:13]: Nopes, will remove

File: app/weather_api/services/met_service.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import asyncio
+
+import aiohttp
+import requests
```
Comment (dymbe) [inline app/weather_api/services/met_service.py:4]: It seems like requests is not used anymore, if that's the case you should remove this import

**Review by dymbe — APPROVED** [review]
LGTM, PR is ready for merge if you fix that final comment

**Pull request merged**